xds/internal/xdsclient/xdslbregistry/xdslbregistry_test.go

```diff
@@ -102,13 +103,22 @@ func (s) TestConvertToServiceConfigSuccess(t *testing.T) {
 					},
 				},
 			},
-			wantConfig: &internalserviceconfig.BalancerConfig{
-				Name: "ring_hash_experimental",
-				Config: &ringhash.LBConfig{
-					MinRingSize: 10,
-					MaxRingSize: 100,
+			wantConfig: `[{"ring_hash_experimental": { "minRingSize": 10, "maxRingSize": 100 }}]`,
+		},
+		{
+			name: "pick_first",
+			policy: &v3clusterpb.LoadBalancingPolicy{
+				Policies: []*v3clusterpb.LoadBalancingPolicy_Policy{
+					{
+						TypedExtensionConfig: &v3corepb.TypedExtensionConfig{
+							TypedConfig: testutils.MarshalAny(&v3pickfirstpb.PickFirst{
+								ShuffleAddressList: true,
+							}),
+						},
+					},
 				},
 			},
+			wantConfig: `[{"pick_first": { "shuffleAddressList": true }}]`,
 		},
```

Comment:
There is no "unset" in the PF proto input.  This is a boolean field.  If it's missing from the struct, it defaults to its zero value of false.  That said, testing that it outputs `false` when the input is `false`/absent-so-zero is worthwhile, so done.

Reply:
Oh. I've seen the bools be wrapped in a proto pointer type. Right, now if it's not explicitly set it gets it's zero (false, not nil because it's a bool value type), and it should make a false config in both cases.

Reply:
> I've seen the proto bools be wrapped in a proto pointer type.

There are wrapper types for use cases where presence is important.  That's not in use here, because presence is not important for this field (missing=default=zero=false).